doc/user/user/pipelines/index.md

```diff
@@ -60,81 +60,50 @@ Click on the "Select" button at the bottom of the pipeline that you'd like to la
-
-### Using pre-saved pipeline parameters
-
-Parameter sets that [have been saved](#saving-pipeline-parameters) can be used on subsequent executions of the pipeline by using the parameters drop-down, just below the pipeline name:
-
-![Named parameters drop-down.](images/parameters-dropdown.png)
+After selecting a pipeline, you will be sent to the pipeline specific launch page where you can modify the pipeline's general information, parameters, reference file, and have a final look at the files that are goig to be used.
+
+1. General Details:
+   ![General Details](images/general-details.png)
+    * Pipeline Name: A custom pipeline name can be entered allowing you to easily identify the specific analysis at a later data.  By default, the name is given the pipeline name followed by the data run (e.g. SNVPhyl__2-15-2021).
+    * Pipeline Description: Any custom text can be entered here.
+    * Email Pipeline Results: There are three options:
+        1. No email: don't send any emails about the pipeline outcome (default)
+        1. On Error: only send an email if the pipeline fails
+        1. On Completion: Send an email if the pipeline fails or when the pipeline successfully completes running.
+1. Parameters: parameters used to run the pipeline.
+    * There are default sets provided by the pipeline developer as well as any that have been saved.
+      ![Default Parameters](images/parameters-default.png)
+    * Parameters can be modified by clicking on the "Modify" button which will expose all available parameters:
+      ![Default Parameters](images/parameters-exposed.png)
+      
+   <blockquote>
+      <p><b>Note</b>: customized pipeline parameters are not validated, please ensure that the values you are entering are valid for the parameter that you are modifying.</p>
+   </blockquote>
+   
+    * If you modify one of the parameters you will be given the option to save the changes for future use.  In the yellow box, just click the "Save template as" button and you will be asked to enter the name for the parameters.  Once saved they can be selected in the dropdown above for future pipelines.
+       ![Save Parameters](images/parameters-modified.png)
+    1. Below these parameters are other settings that can be set (but cannout be saved)
+1. Share Results:
+    1. Some pipeline will allow you to share result back to either the project or sample run.  Checkbox will be visible here if this option is available on this pipeline
+        ![Share Results](images/share-results.png)
+1. Reference Files:
+    1. If the pipeline requires a reference files you will be prompeted to select a reference file from one of the projects or to upload a new one.
```

Comment:
Change `If the pipeline requires a reference file, you will be prompted to select a reference file from one of the projects that you have access to, or to upload a new one.`

Reply:
Updated in 092d339